packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -757,8 +757,8 @@ async function updateS3Templates(context: $TSContext, resourcesToBeUpdated: $TSA
 
   for (const { category, resourceName } of resourcesToBeUpdated) {
```

Comment:
You can the service from here `for(const {category, resouceName, service } of resourceToBeUpdated) {`